recipe-box-backend/src/main/java/com/endava/recipebox/controller/RecipeController.java

```diff
@@ -35,5 +38,14 @@ public ResponseEntity<List<RecipeDTO>> getAllPublicRecipesByName(@RequestParam S
         return ResponseEntity.ok(recipeService.getAllPublicRecipesByName(recipeName));
     }
 
+    @PostMapping("/create")
```

Comment:
For more details read: https://medium.com/@nadinCodeHat/rest-api-naming-conventions-and-best-practices-1c4e781eb6a5, but take into consideration that POST method is self explanatory for creating a resource, so I recommend to delete the create word, for adding a recipe is enough: POST /recipes?userId={userId}.